main.go

```diff
@@ -269,7 +269,8 @@ func schemaFromSQL(driver string) (*internal.Conv, error) {
 		return nil, err
 	}
 	conv := internal.MakeConv()
-	err = ProcessInfoSchema(driver, conv, sourceDB)
+	tableSchema := os.Getenv("MYSQLDATABASE")
```

Comment:
Two issues:

- This function is a generic method for all drivers. We shouldn't read a specific env var for all drivers. 
- `tableSchema` is not clear and might be under a specific context, e.g., MySQL background.

Reply:
If I am using `pgdump` and `postgres`, I should not read `MYSQLDATABASE` env var here. What does `tableSchema` mean for `pgdump` and `postgres`?

Reply:
Yes, the concept of schema is differs significantly between postgres and mysql.

For postgres, it is an extra layer of namespace within a database. By default tables are created using the 'public' schema, but can they can also be defined within a schema (namespace) e.g. myschema.table. When I connect to a postgres server, I only have access to the database I connected to.

In contrast, for mysql, schema is the same as database name. When I connect to a mysql server, I actually have access to all databases on that server, not just to the specific database I used in the connection.

I agree with Hengfeng: I think this has to be handled at the mysql level. And probably deserves a comment along the lines of:

// In MySQL, schema is the same as database name.

Reply:
Agreed. Updated to code and changed parameter to `dbName` as discussed in comments below. 

I am not sure where to place '// In MySQL, schema is the same as database name.' comment. I have added it above the MySQL case in `ProcessInfoSchema` function.